### PR TITLE
Test delayed pokes

### PR DESCRIPTION
**Pull request by brianmcmichael**
Adds tests of delayed pokes. Includes WBTC-B update.

### CODE REVIEW

**Review by gbalabasquer — APPROVED** [review]
LGTM

File: src/MegaPoker.t.sol [hunk 1]
```diff
@@ -349,12 +340,54 @@ contract MegaPokerTest is DSTest, PokingAddresses {
...
+        (, mat) = SpotLike(spotter).ilks("LRC-A");
+        (,, spot,,) = VatLike(vat).ilks("LRC-A");
+        assertTrue(spot != rdiv(value, mat));
         (, mat) = SpotLike(spotter).ilks("GUNIV3DAIUSDC1-A");
         (,, spot,,) = VatLike(vat).ilks("GUNIV3DAIUSDC1-A");
-        assertEq(spot, rdiv(value, mat));
+        assertTrue(spot != rdiv(value, mat));
         (, mat) = SpotLike(spotter).ilks("WBTC-B");
         (,, spot,,) = VatLike(vat).ilks("WBTC-B");
         assertEq(spot, rdiv(value, mat));
```
Comment (naszam) [inline src/MegaPoker.t.sol:362]: just noticed, WBTC-B test block should go in the previous set after WSTETH-A.

Comment (naszam) [reply src/MegaPoker.t.sol:362]: I can open a separate pr eventually, as this is related to #26

Comment (brianmcmichael) [reply src/MegaPoker.t.sol:362]: merge issue. fixed.

File: src/MegaPoker.sol [hunk 2]
```diff
@@ -79,6 +78,7 @@ contract MegaPoker is PokingAddresses {
         (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("WBTC-A")));
         (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("MANA-A")));
         (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("COMP-A")));
+        (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("LINK-A")));
         (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("ETH-B")));
         (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("YFI-A")));
         (ok,) = spotter.call(abi.encodeWithSelector(0x1504460f, bytes32("BAL-A")));
```
Comment (naszam) [inline src/MegaPoker.sol:84]: I think BAL-A can be moved in the 24 block as well as @NiklasKunkel said in the chat

**Comment by brianmcmichael** [comment]
Modified a little from original PR intent.

Now we're doing BAT, ZRX, LRC, BAL, and GUNIV3DAIUSDC1 daily per discord discussion.

**Review by naszam — APPROVED** [review]
Checked:

- `UNIV2AAVEETH` removed form MegaPoker as Offboarding has been completed and confirmed by Nik
- `LINK-A` have been moved to hourly pokes as requested by Nik 
- `BAL-A` have been moved to daily pokes as requested by Nik (Offboarding via Lerp, started with last spell 2021-11-19, executed today)
- `UNIV2LINKETH-A` have been moved to daily pokes as requested by Nik (Offboarding via Lerp, increased target `mat` with the last spell 2021-11-19, executed today)
- Tests Look Good

LGTM (pending final signoff by @NiklasKunkel and OCU to LGTDeploy)

**Pull request merged**